starboard/android/shared/player_components_factory.h

```diff
@@ -215,13 +215,11 @@ class PlayerComponentsFactory : public starboard::shared::starboard::player::
     if (!creation_parameters.audio_mime().empty()) {
       MimeType audio_mime_type(creation_parameters.audio_mime());
       if (!audio_mime_type.is_valid() ||
-          !audio_mime_type.ValidateBoolParameter("enableaudiodevicecallback") ||
           !audio_mime_type.ValidateBoolParameter("audiopassthrough")) {
         return scoped_ptr<PlayerComponents>();
       }
 
-      enable_audio_device_callback =
-          audio_mime_type.GetParamBoolValue("enableaudiodevicecallback", true);
+      enable_audio_device_callback = true;
```

Comment:
We should remove all code related to variable `enable_audio_device_callback` and assume its value is true.

Reply:
Cleanup enable_audio_device_callback and leave it as default value of true.